migrations/00001_initial/index.sql

```diff
@@ -0,0 +1,16 @@
+DROP SCHEMA IF EXISTS emails CASCADE;
+CREATE SCHEMA emails;
+
+DROP TABLE IF EXISTS emails.signer_emails CASCADE;
+CREATE table emails.signer_emails
+(
+    id                SERIAL PRIMARY KEY,
+    chain_id          int                   NOT NULL,
+    email_address     text                  NOT NULL,
+    safe_address      character varying(42) NOT NULL,
+    signer            character varying(42) NOT NULL,
+    verified          boolean               NOT NULL DEFAULT false,
+    verification_code text,
+    sent_on           timestamp with time zone       DEFAULT now() NOT NULL,
```

Comment:
Personally I find this name confusing, as it is not clear what was sent.

Reply:
Yeah good point. Previously it was on the `verification` table so it made more sense due to the context. Will change it to something like `verification_sent_on`

Reply:
Changed in [e335038](https://github.com/safe-global/safe-client-gateway/pull/829/commits/e33503803281ae1ad0a48b4fe29f7f2f9d52adc1)